source/extensions/filters/http/jwt_authn/BUILD

```diff
@@ -71,3 +74,34 @@ envoy_cc_library(
         "//source/extensions/filters/http/common:factory_base_lib",
     ],
 )
+
+envoy_cc_library(
+    name = "verifier_lib",
+    srcs = ["verifier.cc"],
+    hdrs = [
+        "verifier.h",
+    ],
+    external_deps = [
+        "jwt_verify_lib",
+    ],
+    deps = [
+        "@envoy_api//envoy/config/filter/http/jwt_authn/v2alpha:jwt_authn_cc",
+    ],
+)
+
+envoy_cc_library(
+    name = "matcher_lib",
+    srcs = ["matcher.cc"],
+    hdrs = [
+        "matcher.h",
+        "verifier.h",
```

Comment:
Not need to list verifier.h here again since this lib depends on verifier_lib which already listed it.